### ISSUE TITLE
Rubocop didn't detect Rails rules in Rails API project

### ISSUE THREAD

**Issue by BrockBeaudry** [bbatsov/rubocop#5165]
## Expected behavior

Rubocop Rails rules should be detected without adding --rails flag to the rubocop command.

## Actual behavior

Rubocop did not detect that the project was a Rails project.

## Steps to reproduce the problem

Not 100% sure this is because the project was an API project. We generated an example by running this command:

```rails new --skip-yarn --skip-action-mailer --skip-action-cable --skip-sprockets --skip-javascript --skip-turbolinks --skip-test --skip-system-test --api rubocop-example```

Here's an example project that should have a Rails error, but doesn't: https://github.com/BrockBeaudry/rubocop-example

## RuboCop version

```
0.51.0
```

**Comment by pocke** [comment]
RuboCop does not have automatic detection Rails applications. So if you want to run Rails cops, you should add `--rails` option, or modify `.rubocop.yml`.

```yaml
Rails:
  Enabled: true
```
